include/cantera/kinetics/InterfaceKinetics.h

```diff
@@ -240,8 +240,16 @@ class InterfaceKinetics : public Kinetics
      *  \f]
      *
      * @param tstep  Time value to advance the surface coverages
-     */
-    void advanceCoverages(doublereal tstep);
+     * @param rtol   The relative tolerance for the integrator
+     * @param atol   The absolute tolerance for the integrator
+     * @param maxStepSize   The maximum step-size the integrator is allowed to take
+     * @param maxSteps   the maximum number of time-steps the integrator can take
+     *                   before reaching tstep
+     * @param maxErrTestFails   the maximum permissible number of error test failures
+     */
+    void advanceCoverages(doublereal tstep, doublereal rtol=1.e-7,
+                          doublereal atol=1.e-14, doublereal maxStepSize=0,
+                          size_t maxSteps=0, size_t maxErrTestFails=0);
```

Comment:
Instead of using zero as the default value for `maxSteps` and `maxErrTestFails`, it might be more instructive to use the default values from CVODES, which are 500 and 7. For `maxStepSize`, we should document that 0 means no limit is enforced.

Reply:
@speth -- my only concern here is that 500 steps is very low and usually causes a mxsteps exception... Maybe we should use the default from `CVodesIntegrator` (20000)?

Reply:
Ah, good point. I had forgotten that `CVodesIntegrator` overrides the default `maxSteps` of CVODES with its own value. And certainly for the case of `advanceCoverages`, where the most common use case seems to be just advancing to steady state without caring about the intermediate timesteps, maintaining that higher value seems fair.